spec/lib/phobos/listener_spec.rb

```diff
@@ -119,6 +123,16 @@ class TestListenerHandlerWithProducer < TestListenerHandler
     end
   end
 
+  describe 'errors with invalid handlers' do
+    let(:handler_class) { TestBatchListenerHandler }
+    let(:delivery) { 'message' }
+
+    it 'should raise an error if given batch handler in single message mode' do
+      expect { listener.start }.to raise_error(Phobos::InvalidHandlerError,
```

Comment:
not sure if we need this test if we have the proposed solution `batch_inline` as delivery option [#batch_inline]

Reply:
Yep, I'll delete.